deploy/iso/minikube-iso/package/nerdctl-bin/nerdctl-bin.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# nerdctl-bin
+#
+################################################################################
+
+NERDCTL_BIN_VERSION = v0.17.1
```

Comment:
Now we have v0.18.0.

Reply:
@afbjorklund version 0.18 supports `nerdctl build` for local images, may help replacing `docker build`.

Reply:
All of the versions use `buildctl build` behind the scenes, but it can't replace `docker build` without a remote API.

This was why we had to invent the "minikube image" client, so that we could transport the build context to the VM.

The thing that is new in 0.18 is that it allows using containerd also for rootless builds, which we don't support (yet).

So I don't think it makes any difference for the minikube use case, and we need more docs for using nerdctl with k8s.